### PR TITLE
More ROM0 disassembly

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: home/cry.asm [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	pop af
+	jr asm_39c3
+
+PlayCry:: ; 00:39ba
+	push af
+	xor a
+	ld [wc1b9], a
+	ld [wc1ba], a
+	pop af
+asm_39c3: ; 00:39c3
```
Comment (mid-kid) [inline home/cry.asm]: Is this actually used anywhere or is it just your disassembler trying to read opcodes where it shouldn't?

Comment (PikalaxALT) [reply home/cry.asm]: Not sure what you're trying to ask here. These are most certainly the Pokémon cry-playing functions.

Unless you meant to place this comment elsewhere in the file?

Comment (mid-kid) [reply home/cry.asm]: No, I was wondering if that label isn't useless.

Comment (PikalaxALT) [reply home/cry.asm]: I direct you to compare [Crystal's functions.](/pret/pokecrystal/blob/master/home/cry.asm)

**Pull request merged**